wicker/core/storage.py

```diff
@@ -79,9 +102,28 @@ def fetch_file(self, input_path: str, local_prefix: str, timeout_seconds: int =
         :param timeout_seconds: number of seconds till timing out on waiting for the file to be downloaded
         :return: local path to the file on the local file system
         """
-        input_path = os.path.join(input_path, os.path.basename(local_prefix))
-        self.download_with_retries(input_path, local_prefix)
-        return local_prefix
+        local_full_path = os.path.join(local_prefix, os.path.basename(input_path))
+        self.download_with_retries(input_path, local_full_path)
+        return local_full_path
+
+    def put_file(self, input_path: str, target_path: str) -> None:
+        """Put file on local or mounted data storage.
+
+        :param input_path: file path on the local system.
+        :param target_path: file path on local system or mounted drive.
+        """
+        os.makedirs(Path(target_path).parent, exist_ok=True)
+        shutil.copy2(input_path, target_path)
+
+    def put_object(self, object_bytes: bytes, path: str) -> None:
```

Comment:
Super-nit to replace `path: str` with `target_path: str` to match the base class definition of this function.

Reply:
done, much nicer